_episodes/06-working-with-columns.md

```diff
@@ -26,8 +26,6 @@ You can rename a column by opening the drop-down menu at the top of the column t
 ## Sorting data
 You can sort data in OpenRefine by clicking on the drop-down menu for the column you want to sort on, and choosing `Sort`.
 
-Once you have sorted the data, a new `Sort` drop-down menu will be displayed.
-
 Unlike in Excel, 'Sorts' in OpenRefine are temporary - that is, if you remove the `Sort`, the data will go back to its original 'unordered' state. The 'Sort' drop-down menu lets you amend the existing sort (e.g., reverse the sort order), remove existing sorts, and/or make sorts permanent.
 
-You can sort on multiple columns at the same time by adding another sorted column (in the same way).
+To sort multiple columns, click on the drop-down menu for the second column you want to sort and choose `Sort`. In the new window, make sure that `sort by this column alone` is not checked.
```

Comment:
This is definitely an improvement, but I think I'd avoid describing the Sort pop-up as a "new window" - maybe "Sort dialogue" or "Sort pop up"?

Reply:
replaced window with
Sort dialog box